pkg/schemadsl/generator/generator_test.go

```diff
@@ -92,7 +92,9 @@ func TestGenerator(t *testing.T) {
 			"legacy relation",
 			namespace.Namespace("foos/test",
```

Comment:
why not just delete the test case?

Reply:
Because it is still valid for those who might still have schema-less namespaces: we should still convert (so they can fix the problem), and that conversion should still be tested.